pkg/reconciler/ingress/resources/gateway.go

```diff
@@ -72,13 +72,17 @@ func GetHTTPServer(gateway *v1alpha3.Gateway) *v1alpha3.Server {
 }
 
 func belongsToClusterIngress(server *v1alpha3.Server, ia v1alpha1.IngressAccessor) bool {
-	// The format of the portName should be "<clusteringress-name>:<number>".
-	// For example, route-test:0.
+	// The format of the portName should be "<namespace>-<ingress_name>:<number>".
+	// For example, default-routetest:0.
 	portNameSplits := strings.Split(server.Port.Name, ":")
 	if len(portNameSplits) != 2 {
 		return false
 	}
-	return portNameSplits[0] == ia.GetName()
+	// TODO: This should be removed after ClusterIngress codes are cleaned up.
```

Comment:
can namespace even be empty? I don't see any change that adds a namespace.

Reply:
Yes, when ClusterIngress is used, the namespace can be empty. Although the ClusterIngress is not used anymore and Ingress is used, there are still many unit tests which use ClusterIngress with empty namespace.
e.g - https://github.com/knative/serving/blob/master/pkg/reconciler/clusteringress/clusteringress_test.go#L862-L878

I tried to fix the unit tests, but there are too many code changes to include this PR.

Reply:
Test changes should be straightforward, no? Might be a bit noisy, but trivial.

Reply:
Unfortunately it is not straightforward. 
There are many util functions to create ClusterIngress in unit tests, and changing it to Ingress seems not trivial.

Reply:
Well, at least we'll need a issue to track this. Todo's are harder to track.

Reply:
Thank you. Sire, now I opened the ticket https://github.com/knative/serving/issues/5332 to track it.